### PR TITLE
Update dotnet CLI and SDK

### PR DESCRIPTION
**Pull request by zivkan**
Checking tests on 2.2.1xx without dotnet list package tests.

### CODE REVIEW

File: build/config.props [hunk 1]
```diff
@@ -9,8 +9,8 @@
     <SemanticVersion Condition=" '$(SemanticVersion)' == '' ">$(MajorNuGetVersion).$(MinorNuGetVersion).$(PatchNuGetVersion)</SemanticVersion>
     <VsTargetBranch>lab/d16.$(MinorNuGetVersion)stg</VsTargetBranch>
     <VsTargetBranch Condition="'$(IsEscrowMode)' == 'true'">rel/d16.$(MinorNuGetVersion)</VsTargetBranch>
-    <SdkTargetBranch>release/2.1.4xx</SdkTargetBranch>
-    <CliTargetBranch>release/2.1.4xx</CliTargetBranch>
+    <SdkTargetBranch>release/2.2.1xx</SdkTargetBranch>
```
Comment (rohit21agrawal) [inline build/config.props]: wait for this until my change is merged.

Comment (nkolev92) [reply build/config.props]: 2.2.1xx is not correct. 

It's probably gonna be 2.2.2xx at the minimum, as dev ships March, while 2.2.1xx ships November.

**Review by nkolev92 — CHANGES_REQUESTED** [review]

File: build/config.props (hunk 1 above)
Comment (zivkan) [reply build/config.props]: the Download CLI step of the build script fails with a 404 when trying to set the target branch 2.2.2xx

Comment (nkolev92) [reply build/config.props]: The branch doesn't exist yet. 
I'd imagine we should be targetting master. 

Let me pull up some threads to figure out the correct branch.

Comment (nkolev92) [reply build/config.props]: Target master.

Comment (zivkan) [reply build/config.props]: Using master targets a 3.0 preview of the CLI/SDK. Most of the tests that fail are just SHA checking, and there are a few "can't find package" errors, but they seem unrelated to the SDK/CLI, so I think it's good to merge.

**Review by nkolev92 — APPROVED** [review]

**Pull request merged**